{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": "https://github.com/LuminosoInsight/python-ftfy",
    "doc_source_url": null,
    "doc_url": "http://ftfy.readthedocs.io/",
    "home": "https://github.com/LuminosoInsight/python-ftfy/",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/f/ftfy/ftfy-5.3.0.tar.gz",
    "subdirs": [
      "win-64"
    ],
    "summary": "Fixes some problems with Unicode text after the fact",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "5.3.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003",
    "cf201901"
  ],
  "package": "ftfy-5.3.0-py36_0.tar.bz2",
  "repodata": {
    "build": "py36_0",
    "build_number": 0,
    "depends": [
      "python 3.6*",
      "setuptools",
      "wcwidth"
    ],
    "license": "MIT",
    "md5": "86a66aee87b1e4347bcafaa1445ef54b",
    "name": "ftfy",
    "sha256": "f0a76ecdb063bf1756194299018467152bc7dd8fa2cc0d5bca5252c8fbc7275a",
    "size": 70390,
    "subdir": "win-64",
    "version": "5.3.0"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/ftfy-5.3.0-py36_0.tar.bz2"
}